lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -919,9 +919,10 @@ check_terminator({E, _}, [{S, _}|Terminators]) when
 check_terminator({E, {Line, _, _}}, [{Start, {StartLine, _, _}}|_]) when
     E == 'end'; E == ')'; E == ']'; E == '}'; E == '>>' ->
   End = terminator(Start),
-  Message = io_lib:format("\"~ts\" starting at line ~B is missing terminator \"~ts\". "
-                          "Unexpected token: ", [Start, StartLine, End]),
-  {error, {Line, Message, atom_to_list(E)}};
+  MessagePrefix = "Unexpected token: \"",
```

Comment:
Perfect! Just one very tiny nitpick (sorry), our error messages start with lowercase, so this would need to be "unexpected token: \"". Can you please amend and i will merge it straight-away! Thank you so much!